src/app/components/vertical_navbar.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { Home, CalendarDays, Clock }  from "lucide-react";
+import Link from "next/link";
+
+export default function Vertical_Navbar_Component() {
+  return (
+    <>
+    <ul className="m-0 h-max p-0 pb-96 w-10 bg-stone-400">
```

Comment:
When I looked at the navbar, it did not expand to the bottom of the screen. I changed the padding to pb-full and the height to h-screen and it worked out

Reply:
Ah, sorry about that!